### ISSUE TITLE
How to run python example on Linux

### ISSUE THREAD

**Issue by 4SkyNet** [nadavbh12/Retro-Learning-Environment#5]
Hi, I've got some troubles with the given example in:
> ~/doc/examples

I try to run:
> python_example.py path_to_game_rom

But it fails with issues wrt argv[2], where I should provide some core_file (*.so)

I try again:
> python_example.py path_to_game_rom path_to_librle_c.so

But it fails again with this error:
> Failed to load symbol 'retro_init'': ../../rle_python_interface/librle_c.so: undefined symbol: retro_init

It seems to that there should be some corrections wrt symbols
(it can be linux specific as previous commit to `setup.py` from my buddy)

**Comment by nadavbh12** [comment]
The .so file should be of the core (Atari or Snes).
For snes it should be in snes9x2010/snes9x2010-libretro.so.

I'll add the instructions to help avoid confusion.

**Comment by 4SkyNet** [comment]
Thx, it works!

**Comment by nadavbh12** [comment]
Sorry for the delayed response, didn't notice your comment earlier.
Please give it another try and let me know if you still have this issue.

**Comment by tohnperfect** [comment]
I also got the same error with v1.0.2 .

**Comment by nadavbh12** [comment]
Please make sure that the name of the rom on your file system is identical to that in the [rom file](https://github.com/nadavbh12/Retro-Learning-Environment/blob/master/src/games/supported/SuperMarioWorld.hpp#L47).

**Comment by tohnperfect** [comment]
Thank you very much @nadavbh12,  I am able run the example code now.

Supposedly, tutorial, more examples, and how to use the RLE can be find in the ALE?

**Comment by nadavbh12** [comment]
You're welcome.
Not sure what you're asking. Could you rephrase that?

**Comment by tohnperfect** [comment]
Where I could find more examples and tutorial of your library? Should I
start with the ALE?

Thank you for making this great library.

**Comment by nadavbh12** [comment]
Basically any repository you find online which uses ALE can easily be replaced by RLE.
You can check out [this](https://github.com/nadavbh12/deep_q_rl) fork of deep_q_rl to run DQN with RLE.

**Comment by tohnperfect** [comment]
Great! Thank you very much for the pointer.
